packages/forms/test/reactive_integration_spec.ts

```diff
@@ -2733,6 +2733,48 @@ const ValueAccessorB = createControlValueAccessor('[cva-b]');
             expect(form.controls.pin.errors).toEqual({max: {max: -10, actual: 0}});
           });
         });
+
+        it('should fire registerOnValidatorChange for formGroups', () => {
+          let registeronValidatorChangeFired = 0;
+
+          @Directive({
+            selector: '[ng-empty-custom-validator]',
```

Comment:
Nit (a bit shorter):
```suggestion
            selector: '[ng-noop-validator]',
```